.github/workflows/link-check.yml

```diff
@@ -0,0 +1,62 @@
+name: Link Check
+
+on:
+  schedule:
+    - cron: "00 8 * * *" # 0800 UTC is 0400 AM EDT
```

Comment:
@david-waltermire-nist set in repo clean up to 1500 PM ET to 0400 AM ET per final adhoc review this afternoon. Ready for review!